src/quo2/components/notifications/activity_logs_photos/view.cljs

```diff
@@ -0,0 +1,14 @@
+(ns quo2.components.notifications.activity-logs-photos.view
+  (:require [react-native.core :as rn]
+            [quo2.components.notifications.activity-logs-photos.style :as style]))
+
+(defn view
+  [photos]
```

Comment:
nit: should we use a map for first arg as a standard for all our quo2 components? It's quite react style to do that 👍